Add Pause Menu In game

NicolasTre opened the pull request:
I add a Pause Menu in the game with 2 button ( Resume and Main Menu )

NicolasTre committed:
Add Pause Menu In game

I add a Pause Menu in the game with 2 button ( Resume and Main Menu )

NicolasTre committed:
Add Pause Menu In game

I add a Pause Menu in the game with 2 button ( Resume and Main Menu )

Zormzy reviewed (changes requested):
Bad use of input.
Need to be new input system for pause menu trigger.

NicolasTre committed:
Using NewInputSystem

Using the new input System and modification of the script menupause with the new inputsystem

NicolasTre committed:
Merge branch 'Nicolas' of https://github.com/Zormzy/Pigeon-s-Legacy into Nicolas

Zormzy reviewed (approved):
All good.
